docs/benchmarks/test-results-summary.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance Benchmarks and Test Reports
+---
+
+
+## Introduction
+This page summarizes key performance metrics for nwaku and provides links to detailed test reports.
+
+> ## Quick reference
+>
+> - **10%** decrease in bandwidth usage in larger networks (1000 vs 2000 relay nodes)
+> - **0.4s** average, **5.5s** max message propogation latency (2000 relay nodes)
```

Comment:
```suggestion
> - **0.4s** average, **5.5s** max message propagation latency (2000 relay nodes)
```
I'd just be explicit here about what we we mean by "message propagation latency". In other words, "0.4s on average for a message to propagate to 100% of nodes in topologies of up to 2000 nodes".